New release on Rolling

These packages are broken on `rolling`. https://build.ros2.org/view/Rbin_uN64/job/Rbin_uN64__rclc__ubuntu_noble_amd64__binary/168/console

I try to make release but I don't have permissions. I already push the changes in the Changelog. Someone needs to execute:

```
bloom-release --rosdistro rolling --track rolling rclc
```

@ahcorde thanks for pointing that out. 
```
make[1]: *** [debian/rules:33: override_dh_auto_configure] Error 2
make[1]: Leaving directory '/tmp/binarydeb/ros-rolling-rclc-6.2.0'
make: *** [debian/rules:27: build] Error 2
```
in the console output, I could only see, that `rclc` package created a problem. But no specific error message. Sure, I can make a new release for rolling and create a new bloom-release. 

I would do now the following:
- merge the branch for bloom-release-6.2.1 from last year: https://github.com/ros2/rclc/pull/448
- make new release 6.2.2 (to reflect the current tip-of-branch)

The error message is this one 

```
10:33:46 CMake Error at CMakeLists.txt:82 (ament_target_dependencies):
10:33:46   Unknown CMake command "ament_target_dependencies".
10:33:46 
10:33:46 
10:33:46 -- Configuring incomplete, errors occurred!
10:33:46 	cd .obj-x86_64-linux-gnu && tail -v -n \+0 CMakeCache.txt
```

and it was fixed here https://github.com/ros2/rclc/pull/419

Thank you for the release!

--- a/rclc/CHANGELOG.rst
+++ b/rclc/CHANGELOG.rst
@@ -2,8 +2,8 @@
 Changelog for package rclc
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Forthcoming
------------
+6.2.1 (2025-05-22)
+------------------
 * Use target_link_libraries instead of ament_target_dependencies (#419)
 * Do not return error when rcl_send_response timeout. (#408)
 

--- a/rclc/CHANGELOG.rst
+++ b/rclc/CHANGELOG.rst
@@ -2,6 +2,11 @@
 Changelog for package rclc
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Use target_link_libraries instead of ament_target_dependencies (#419)
+* Do not return error when rcl_send_response timeout. (#408)
+
 6.2.0 (2024-10-15)
 ------------------
 * added CI status for iron builds (#377)

--- a/rclc/package.xml
+++ b/rclc/package.xml
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>rclc</name>
-  <version>6.2.0</version>
+  <version>6.2.1</version>
   <description>The ROS client library in C.</description>
   <maintainer email="jan.staschulat@de.bosch.com">Jan Staschulat</maintainer>
   <maintainer email="pablogarrido@eprosima.com">Pablo Garrido</maintainer>

--- a/rclc_examples/CHANGELOG.rst
+++ b/rclc_examples/CHANGELOG.rst
@@ -2,6 +2,10 @@
 Changelog for package rclc_examples
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Use target_link_libraries instead of ament_target_dependencies (#419)
+
 6.2.0 (2024-10-15)
 ------------------
 * fix rclc_example: memory leaking in msg.data allocation (backport #386) (#387)

--- a/rclc_examples/CHANGELOG.rst
+++ b/rclc_examples/CHANGELOG.rst
@@ -2,8 +2,8 @@
 Changelog for package rclc_examples
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Forthcoming
------------
+6.2.1 (2025-05-22)
+------------------
 * Use target_link_libraries instead of ament_target_dependencies (#419)
 
 6.2.0 (2024-10-15)

--- a/rclc_examples/package.xml
+++ b/rclc_examples/package.xml
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>rclc_examples</name>
-  <version>6.2.0</version>
+  <version>6.2.1</version>
   <description>Example of using rclc_executor</description>
   <maintainer email="jan.staschulat@de.bosch.com">Jan Staschulat</maintainer>
 

--- a/rclc_lifecycle/CHANGELOG.rst
+++ b/rclc_lifecycle/CHANGELOG.rst
@@ -2,8 +2,8 @@
 Changelog for package rclc_lifecycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Forthcoming
------------
+6.2.1 (2025-05-22)
+------------------
 * Use target_link_libraries instead of ament_target_dependencies (#419)
 
 6.2.0 (2024-10-15)

--- a/rclc_lifecycle/CHANGELOG.rst
+++ b/rclc_lifecycle/CHANGELOG.rst
@@ -2,6 +2,10 @@
 Changelog for package rclc_lifecycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Use target_link_libraries instead of ament_target_dependencies (#419)
+
 6.2.0 (2024-10-15)
 ------------------
 * no changes

--- a/rclc_lifecycle/package.xml
+++ b/rclc_lifecycle/package.xml
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>rclc_lifecycle</name>
-  <version>6.2.0</version>
+  <version>6.2.1</version>
   <description>rclc lifecycle convenience methods.</description>
   <maintainer email="jan.staschulat@de.bosch.com">Jan Staschulat</maintainer>
 

--- a/rclc_parameter/CHANGELOG.rst
+++ b/rclc_parameter/CHANGELOG.rst
@@ -2,8 +2,8 @@
 Changelog for package rclc_parameter
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Forthcoming
------------
+6.2.1 (2025-05-22)
+------------------
 * Use target_link_libraries instead of ament_target_dependencies (#419)
 
 6.2.0 (2024-10-15)

--- a/rclc_parameter/CHANGELOG.rst
+++ b/rclc_parameter/CHANGELOG.rst
@@ -2,6 +2,10 @@
 Changelog for package rclc_parameter
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Use target_link_libraries instead of ament_target_dependencies (#419)
+
 6.2.0 (2024-10-15)
 ------------------
 * Use fully qualified node name in parameter events (#402)

--- a/rclc_parameter/package.xml
+++ b/rclc_parameter/package.xml
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>rclc_parameter</name>
-  <version>6.2.0</version>
+  <version>6.2.1</version>
   <description>Parameter server implementation for micro-ROS nodes</description>
   <maintainer email="antoniocuadros@eprosima.com">Antonio Cuadros</maintainer>
 

